src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuegenericletter/IssueGenericLetterAboutToStartHandlerTest.java

```diff
@@ -145,4 +132,21 @@ void givenDocumentsAreExist_thenShouldReturnDocumentsList() {
         assertEquals(1, documentSelection.size());
         assertEquals(2, documentSelection.get(0).getValue().getDocumentsList().getListItems().size());
     }
+
+    @Test
+    void givenDocumentEdited_ThenAddToListOptions() {
+        when(callback.getEvent()).thenReturn(ISSUE_GENERIC_LETTER);
+        when(callback.getCaseDetails()).thenReturn(caseDetails);
+        when(caseDetails.getCaseData()).thenReturn(caseData);
+
+        DocumentLink editDoc = DocumentLink.builder().documentFilename("EditedSscsDocument").documentUrl("EditedUrl").build();
+        SscsDocument sscsDocument = new SscsDocument(SscsDocumentDetails.builder().documentFileName("SscsDocument").editedDocumentLink(editDoc).build());
+        caseData.setSscsDocument(Collections.singletonList(sscsDocument));
+
+        var result = handler.handle(ABOUT_TO_START, callback, USER_AUTHORISATION);
+
+        List<CcdValue<DocumentSelectionDetails>> documentSelection = result.getData().getDocumentSelection();
+        assertEquals(1, documentSelection.size());
+        assertEquals(3, documentSelection.get(0).getValue().getDocumentsList().getListItems().size());
```

Comment:
are you able to assert that the same document you've added shows up in the result?